src/components/TransactionsTable/style.scss

```diff
@@ -216,8 +209,58 @@
       text-align: center;
     }
 
+    .spinner_container {
+      margin-bottom: 2000px;
+    }
+
     @media all and (max-width: $screen-tablet) {
       @include rem(padding, 0 24px 0 0);
     }
   }
+
+  .sticky-row {
+    background: $white;
+    position: fixed;
+    top: -160px;
+    left: 0;
+    right: 0;
```

Comment:
One of the style (left or right) is redundant

Reply:
Both are required to put this block in the middle https://www.w3.org/TR/css-position-3/#size-and-position-details